src/mono/mono/metadata/appdomain.c

```diff
@@ -2480,7 +2480,7 @@ mono_domain_assembly_preload (MonoAssemblyLoadContext *alc,
 
 		char *base_dir = get_app_context_base_directory (error);
 		search_path [0] = base_dir;
-		mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_ASSEMBLY, "Domain %s (%p) ApplicationBase is %s", domain->friendly_name, domain, base_dir);
+		mono_trace (G_LOG_LEVEL_DEBUG, MONO_TRACE_ASSEMBLY, "Domain (%p) ApplicationBase is %s", domain, base_dir);
 
```

Comment:
Why is this needed ?

Reply:
On netcore we only have one domain and it's named 'mono', so this doesn't actually offer any useful information and looks dumb in the logs. Note that this log is in netcore-specific code and I haven't touched the legacy equivalent. If you think there's a reason to keep it I'm happy to, but otherwise might as well clean it up.